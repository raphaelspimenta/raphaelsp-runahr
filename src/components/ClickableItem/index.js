import React from 'react'
import { Box, Text } from 'grommet'
import { BarChart } from 'grommet-icons'

const ClickableItem = ({ onClick, text }) => (
  <Box
    direction="row-responsive"
    justify="center"
    align="center"
    pad="xlarge"
    gap="medium"
  >
    <Box
      pad="large"
      align="center"
      background={{ color: 'brand', opacity: 'strong' }}
      round
      gap="small"
      onClick={onClick}
    >
      <BarChart size="large" />
      <Text color="light-2">{text}</Text>
    </Box>
  </Box>
)

export default ClickableItem
