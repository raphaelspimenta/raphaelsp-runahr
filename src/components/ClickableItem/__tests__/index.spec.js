import React from 'react'
import { shallow } from 'enzyme'
import ClickableItem from '../'

describe('ClickableItem', () => {
  it('should render', () => {
    const component = shallow(<ClickableItem text="test" onClick={() => {}} />)
    expect(component).toMatchSnapshot()
  })
})
