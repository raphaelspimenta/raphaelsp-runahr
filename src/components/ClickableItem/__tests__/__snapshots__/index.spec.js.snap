// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClickableItem should render 1`] = `
<Box
  align="center"
  direction="row-responsive"
  gap="medium"
  justify="center"
  pad="xlarge"
>
  <Box
    align="center"
    background={
      Object {
        "color": "brand",
        "opacity": "strong",
      }
    }
    gap="small"
    onClick={[Function]}
    pad="large"
    round={true}
  >
    <BarChart
      size="large"
    />
    <Text
      color="light-2"
      level={1}
    >
      test
    </Text>
  </Box>
</Box>
`;
